nirc_ehr/resources/referenceStudy/study/datasets/datasets_metadata.xml

```diff
@@ -100,6 +100,9 @@
             <column columnName="arrivalProtocol">
                 <datatype>varchar</datatype>
             </column>
+            <column columnName="acquisitionType">
```

Comment:
Looks like this needs formatting/lookup metadata for the grid and form. Probably should be added to default view of arrival.

Reply:
done